packages/react-composites/tests/browser/call/CallComposite.test.ts-snapshots/local device buttons should show tooltips on hover

```diff
@@ -0,0 +1 @@
+{"getPage-parent rerender":3,"-parent rerender":378,"selectLocalPreview-parent rerender":87,"getLocalMicrophoneEnabled-parent rerender":87,"-return state changed":72,"selectLocalPreview-return state changed":12}
```

Comment:
Tested this for several times - it looks pretty stable while I ran 10+ test updates